### PR TITLE
Some changes in fa_IR

### PR DESCRIPTION
**Pull request by Saeid-Khaleghi**
In the Persian language we use these added words to make dummy texts. I suggest you to add this file to fa_IR folder.

### CODE REVIEW

**Comment by Saeid-Khaleghi** [comment]
All of English colors names have an equal Persian name that I have added!

File: src/Faker/Factory.php [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Factory
 {
     const DEFAULT_LOCALE = 'en_US';
 
-    protected static $defaultProviders = array('Address', 'Barcode', 'Biased', 'Color', 'Company', 'DateTime', 'File', 'HtmlLorem', 'Image', 'Internet', 'Lorem', 'Miscellaneous', 'Payment', 'Person', 'PhoneNumber', 'Text', 'UserAgent', 'Uuid');
+    protected static $defaultProviders = array('Address', 'Barcode', 'Biased', 'Color', 'Company', 'DateTime', 'File', 'HtmlLorem', 'Image', 'Internet', 'Lorem', 'Miscellaneous', 'Payment', 'Person', 'Pet', 'PhoneNumber', 'Text', 'UserAgent', 'Uuid');
```
Comment (ppelgrims) [inline src/Faker/Factory.php]: I reckon this belongs in a separate package, which you can add to the readme in a separate PR.

File: src/Faker/Provider/Pet.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
+<?php
```
Comment (ppelgrims) [inline src/Faker/Provider/Pet.php]: See previous comment

File: src/Faker/Provider/fa_IR/Lorem.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Faker\Provider\fa_IR;
+
+class Lorem extends \Faker\Provider\Lorem
+{
+    protected static $wordList = array(
+        'آنچنان','ابزارهای','ارائه','از','اساساً','است','استفاده','اصلی','امید','اهل','ای','ایپسوم','ایجاد','این','آینده','با',
```
Comment (ppelgrims) [inline src/Faker/Provider/fa_IR/Lorem.php]: Please provide a source for this.

File: src/Faker/Provider/fa_IR/Color.php [hunk 4]
```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Faker\Provider;
+
+/**
+ * @author lsv
+ */
+class Color extends Base
```
Comment (ppelgrims) [inline src/Faker/Provider/fa_IR/Color.php]: Please extend `\Faker\Provider\Color`, that way you don't have to repeat most methods.
And have a look at the [color provider from ar_SA](https://github.com/fzaninotto/Faker/blob/master/src/Faker/Provider/ar_SA/Color.php), he included a source as well.

File: src/Faker/Provider/fa_IR/Pet.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Faker\Provider\fa_IR;
+
+class Pet extends \Faker\Provider\Pet
```
Comment (ppelgrims) [inline src/Faker/Provider/fa_IR/Pet.php]: This should go in your separate package as well.

**Comment by pimjansen** [comment]
@Saeid-Khaleghi Bump

**Comment by pimjansen** [comment]
I am going to close this PR as stale. If this still needs to be merged, please feel free to reopen it.
Thanks for contributing

**Pull request closed without merging**